projects/fsastorefrontlib/src/occ/occ-models/occ.models.ts

```diff
@@ -81,10 +81,17 @@ export interface OBOConsentList {
 
 export interface OBOCustomerList {
   entries: FSUser[];
-  pagination: PaginationModel;
+  pagination: FSPaginationModel;
```

Comment:
I made a new interface `FSPaginationModel ` since the structure of data we get from the back doesn't correspond to `PaginationModel `